src/authInfo.ts

```diff
@@ -832,6 +839,12 @@ export class AuthInfo extends AsyncCreatable<AuthInfo.Options> {
 
       if (this.isTokenOptions(options)) {
         authConfig = options;
+        const userInfo = await this.retrieveUsername(
+          ensureString(options.instanceUrl),
+          ensureString(options.instanceUrl)
```

Comment:
should this be called with `options.access_token`? It's hard to tell in the UI, but it looks like it's be called with the same param twice

Reply:
Yes, good catch, thank you!